File: C++/abstractclass_shape.cpp
#include<iostream>
using namespace std;
class Shape
{
	public:
		virtual void draw()=0;
};
class Rectangle:public Shape
{
	public:
		void draw()
		{
			cout<<"draw from rectangle:";
		}
};
int main()
{
	Rectangle r;
	r.draw();
	return 0;
}

File: Csharp_programs/Oops/abstract_class_shape.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace abstract_class_shape
{
    abstract class Shape
    {
        public abstract void draw();
    }
    class Rectangle:Shape
    {
        public override void draw()
        {
            Console.WriteLine("Drawing Rectangle....");
        }
    }
        
    
    internal class Program
    {
        static void Main(string[] args)
        {
            Shape s= new Rectangle();
            s.draw();

            Console.ReadKey();
        }
    }
}
